action/action-dsl/src/test/resources/actions.groovy

```diff
@@ -21,7 +21,7 @@ rule('rule') {
 
 action('action') {
     description 'action description'
-    tasks {
+    modifications {
```

Comment:
@flo-dup you should ensure backward compatibility, otherwise we are going to break a lot of project code.

Reply:
The problem is that Groovy script could be considered as data for some project and not only code. So we can assume to breaj Java compat because it will only impact devs when upgrading powsybl version. But for Groovy script it is a little bit different I think.

Reply:
Still to be done!